### PR TITLE
Move to plugins-core 0.7.2

### PR DESCRIPTION
**Pull request by loosebazooka**
- also changes extraFilesDirectory -> extraFilesDirectory

### CODE REVIEW

File: src/test/java/com/google/cloud/tools/gradle/appengine/AppEnginePluginTest.java [hunk 1]
```diff
@@ -61,6 +62,7 @@ public void testCheckGradleVersion_fail() throws IOException {
                   + AppEngineCorePluginConfiguration.GRADLE_MIN_VERSION
                   + " or higher."));
     }
+    JavaVersion.current().isJava9Compatible();
```
Comment (chanseokoh) [inline src/test/java/com/google/cloud/tools/gradle/appengine/AppEnginePluginTest.java]: What is this for? Why?

Comment (loosebazooka) [reply src/test/java/com/google/cloud/tools/gradle/appengine/AppEnginePluginTest.java]: yeah that shouldn't be in there, must have been around from some old thing I was testing. I'll remove it

File: src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlExtension.java [hunk 2]
```diff
@@ -80,19 +83,20 @@ public void setStagingDirectory(Object stagingDirectory) {
...
   @Optional
   @InputDirectory
-  public File getExtraFilesDirectory() {
+  public List<File> getExtraFilesDirectory() {
     return extraFilesDirectory;
   }
 
   public void setExtraFilesDirectory(Object extraFilesDirectory) {
-    this.extraFilesDirectory = project.file(extraFilesDirectory);
+    this.extraFilesDirectory = new ArrayList<>(project.files(extraFilesDirectory).getFiles());
```
[the remark this answers is not in the document]
Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlExtension.java]: Also used in run extentions:
- https://github.com/GoogleCloudPlatform/app-gradle-plugin/blob/master/src/main/java/com/google/cloud/tools/gradle/appengine/standard/RunExtension.java#L272
- https://github.com/GoogleCloudPlatform/app-gradle-plugin/blob/master/src/main/java/com/google/cloud/tools/gradle/appengine/standard/RunExtension.java#L342

[the remark this answers is not in the document]
Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlExtension.java]: @chanseokoh actually you're right, this is incorrect. I didn't account for the case that the user never configured this value at all (and there is no default like run).

[the remark this answers is not in the document]
Comment (loosebazooka) [reply src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/StageAppYamlExtension.java]: will push a followup PR.

**Pull request merged**